### PR TITLE
ignore grammarly-btn tag and a couple noisy attributes

### PR DESCRIPTION
**Pull request by davidsbailey**
based on html removal events since March 25 in the report at:  https://insights.newrelic.com/accounts/501463/query?query=SELECT%20currentUrl,channelId,removedHtml,safeHtml,unsafeHtml,sanitizationTarget,userAgentName,userAgentOS,userAgentVersion,city,regionCode,countryCode%20from%20PageAction%20where%20actionName%20%3D%20%27SanitizedLevelHtml%27%20since%204%20weeks%20ago%20limit%201000

### CODE REVIEW

File: apps/src/applab/sanitizeHtml.js [hunk 1]
```diff
@@ -51,10 +51,16 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
   var ignoredAttributes = [
     'pmbx_context',   // Used by Chrome plugins such as Bitdefender Wallet.
     'kl_vkbd_parsed', // Origin unknown. Assumed to be a plugin of some kind.
+    'kl_virtual_keyboard_secure_input', // Origin unknown.
```
Comment (islemaster) [inline apps/src/applab/sanitizeHtml.js]: I suspect these two `kl_` keys come from the [Kapersky Labs Password Manager](http://www.kaspersky.com/password-manager) - virtual keyboard being their utility for typing passwords while bypassing keyloggers.

File: apps/src/applab/sanitizeHtml.js [hunk 2]
```diff
@@ -74,6 +80,9 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
           attribs: attribs
         };
       }
+    },
+    exclusiveFilter: function (element) {
+      return (ignoredTags.indexOf(element.tag) !== -1);
```
Comment (islemaster) [inline apps/src/applab/sanitizeHtml.js:85]: [Documentation for `exclusiveFilter` option in the sanitize-html package](https://www.npmjs.com/package/sanitize-html#filters).

Comment (islemaster) [inline apps/src/applab/sanitizeHtml.js:85]: Does this filter need to be case-insensitive, or is element.tag guaranteed to be lowercase?  Might be worth covering with a test.

Comment (davidsbailey) [inline apps/src/applab/sanitizeHtml.js:85]: It looks like `element.tag` is always lower case, based on a quick test. So are attributes.

**Comment by islemaster** [comment]
LGTM

**Pull request merged**